kubectx

```diff
@@ -94,19 +97,31 @@ save_context() {
   saved="$(read_context)"
 
   if [[ "${saved}" != "${1}" ]]; then
-    printf %s "${1}" > "${KUBECTX}"
+    printf %s "${1}" > "${KUBECTX_PREVIOUS_VALUE}"
   fi
 }
 
 switch_context() {
   $KUBECTL config use-context "${1}"
 }
 
+context_list_count() {
+  if [[ -f ${CONTEXT_LIST_CACHE} ]]
+  then
+    echo $(($(cat ${CONTEXT_LIST_CACHE} | wc -l) + 2))
```

Comment:
the +2 doesn't seem to work well when there is 1 context. when you specify --height=3, there's an empty line at the top of the screen.

Try `FZF_DEFAULT_COMMAND=date fzf --ansi --height=3`. That said, `2` or `1` exposes the same problem as well. So could this be a fzf bug?